README.md

```diff
@@ -30,4 +30,23 @@ a suitable value, an `envvar.ValueError` is thrown:
 
     ValueError: Value of process.env["HTTP_MAX_SOCKETS"] does not represent a number
 
-That's all there is to it.
+### `envvar.enum`
+
+This is similar to `envvar.string`, but with constraints. There may be a small
+number of valid values for a given environment variable. For example:
+
+```javascript
+const NODE_ENV = envvar.enum('NODE_ENV', ['development', 'staging', 'production']);
+```
+
+This states that `process.env.NODE_ENV` must be set to `development`,
+`staging`, or `production`.
+
+A default value may be provided:
+
+```javascript
+const NODE_ENV = envvar.enum('NODE_ENV', ['development', 'staging', 'production'], 'production');
```

Comment:
this example is very relevant for us, I've seen everything from `dev` to `development` to `test`, `testing`, etc.